enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresInteractionTestBase.java

```diff
@@ -871,6 +870,23 @@ public void shouldNotListUsersForRoleIfNotAdmin() throws Exception
         testFailListRoleUsers( schemaSubject, ADMIN, PERMISSION_DENIED );
     }
 
+    //---------- clearing authentication cache -----------
+
+    @Test
+    public void shouldClearAuthenticationCacheIfAdmin() throws Exception
```

Comment:
`shouldHaveClearAuthenticationCacheProc`? It does not actually check the caching